Core/Core/Helpers/Http.cs

```diff
@@ -30,16 +30,10 @@ public static AsyncRetryPolicy<HttpResponseMessage> HttpAsyncPolicy(IEnumerable<
       .HandleTransientHttpError()
       .WaitAndRetryAsync(
         delay ?? DefaultDelay(),
-        (ex, timeSpan, retryAttempt, context) => {
-          //context.Remove("retryCount");
-          //context.Add("retryCount", retryAttempt);
-          //Log.Information(
-          //  ex.Exception,
-          //  "The http request failed with {exceptionType} exception retrying after {cooldown} milliseconds. This is retry attempt {retryAttempt}",
-          //  ex.GetType().Name,
-          //  timeSpan.TotalSeconds * 1000,
-          //  retryAttempt
-          //);
+        (ex, timeSpan, retryAttempt, context) =>
+        {
+          context.Remove("retryCount");
+          context.Add("retryCount", retryAttempt);
```

Comment:
this is to update?

Reply:
yep, set the new retry count to be the latest number of retry attempts